sklearn/multioutput.py

```diff
@@ -6,6 +6,8 @@
 extends single output estimators to multioutput estimators.
 """
 
+from abc import ABCMeta, abstractmethod
+
```

Comment:
This is being imported before the `# Authors` section. I'm guessing `isort` needs an empty line above `from numbers import Integral` to function correctly.